CHANGELOG.md

```diff
@@ -2975,7 +2975,7 @@ https://iojs.org/api/tls.html
 - Added async session storage events.
 - Added async SNI callback.
 - Added multi-key server support (for example, ECDSA+RSA server).
-- Added optional callback to `checkServerIdentity` for manual certificate validation in user-land.
+- Added optional callback to `checkServerIdentity` for manual certificate validation in userland.
```

Comment:
It's unnecessary but harmless, in my opinion. It's in accord with the idea of standardizing the references - the more people see "userland" instead of "user-land" the more it catches on.